### DISCUSSION TITLE
Promotion discount display on products page

### DISCUSSION THREAD

**Discussion by jaymar-webriq** [swellstores/community#270] [Ideas]
Good day,
We want to display the discounted price on products page, we have multiple promotion discount with multiple customer, how to implement this kind? 
We are using API backend and a custom storefront base on reactjs

**Comment by sags95** [comment]
Promotions apply on carts rather than on products themselves. We do have a `sale_price` field on products, but this is controlled outside of the promotion price. Are you using swell-js or graphql on your storefront or everything is routed through the Backend API?

**Reply by igoraugustynski** [reply]
Hey @sags95, I have a similar issue. Is there a way to show available promotions on a product page using swell-js?

**Reply by sags95** [reply]
@igoraugustynski  Unfortunately this is not possible with swell-js, but it's a feature that I can pass on to the team as feedback.
